thunder/tests/test_cudnn_executor.py

```diff
@@ -186,15 +188,10 @@ def test_cudnn_vs_torch_consistency(op, device, dtype, *_):
             return result
 
 
-# NOTE Scaled_Dot_Product_Efficient_Attention_Backward does not support fp64 dtypes
-# RuntimeError: Only fp32, half & bf16 supported at the moment
-@ops(
```

Comment:
I didn't manage to add an extra parameter (e.g. `may_cat_grad_qkv`) to the `ops` instantiation. So I reworked (and happened to simplify) the parameterization part of the test.